squid/datadog_checks/squid/data/conf.yaml.example

```diff
@@ -206,4 +206,25 @@ instances:
     ## Whether or not to persist cookies and use connection pooling for increased performance.
     #
     # persist_connections: false
-    
\ No newline at end of file
+    
+    ## Log Section (Available for Agent >=6.0)
```

Comment:
```suggestion
## Log Section (Available for Agent >=6.0)
```
This should be less indented (same for the rest of the file)

Reply:
Leading spaces removed from the whole block, thanks for pointing this out.